docs/manual/src/docs/asciidoc/_includes/reactive/oauth2/access-token.adoc

```diff
@@ -33,3 +33,46 @@ SecurityWebFilterChain configure(ServerHttpSecurity http) throws Exception {
 ----
 
 You can now leverage Spring Security's <<webclient>> or <<webflux-roac,@RegisteredOAuth2AuthorizedClient>> support to obtain and use the access token.
+
+== BearerTokenResolver
+
+With interface BearerTokenResolver you can provide a strategy to resolve a bearer token.
+
+The interface provides the next method:
```

Comment:
Instead of showing the interface, could you show example usage? Specifically, I'm thinking it would be nice for the documentation to show how to do form-based bearer tokens and using a custom header name.

Reply:
@jzheaux , 1) Can you explain more about it? is It like Loosely Coupling? When we take parameters and generate a token?  2) We can provide a link to code with HeaderBearerTokenResolver.

Reply:
Good questions. 

For the first, there is a setting, `DefaultBearerTokenResolver#setAllowFormEncodedBodyParameter`. The spec allows for passing the bearer token [via a form parameter](https://tools.ietf.org/html/rfc6750#section-2.2) instead of a header.

For the second (and in general), I recommend that the code be inlined, so the reader can see example usage in the context of your explanation. For example, you can imagine documentation written like this:

> Or, we may need to use a custom header name, like when using Google Cloud's IAP Proxy authentication:
> ```java
>    http
>        .oauth2ResourceServer()
>             .bearerTokenResolver(new HeaderBearerTokenResolver("x-goog-iap-jwt-assertion"))
>  ```